### ISSUE TITLE
Issue with line endings

### ISSUE THREAD

**Issue by boorad** [Venryx/zalc#1]
I tried to run `zalc` on a Mac, and saw the following:

    ❯ zalc publish
    env: node\r: No such file or directory

The temporary solution was here: https://stackoverflow.com/a/30349952/598628, and so I ran:

    ❯ dos2unix /Users/brad/.config/yarn/global/node_modules/zalc/Dist/Bin.js
    dos2unix: converting file /Users/brad/.config/yarn/global/node_modules/zalc/Dist/Bin.js to Unix format...

But it would be nice to not have to do this.  Would different line-endings still work on Windows?

**Comment by Venryx** [comment]
Good point; I'm assuming leaving out the "\r" (carriage return character) would be just fine on Windows, so will try to make that change soon (added to personal todo list).
